.github/workflows/publish-action.yml

```diff
@@ -10,12 +10,14 @@ on:
 
 jobs:
   release:
-    name: GitHub Release
+    name: Release
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
       - name: GitHub semver release
         uses: vivantehealth/github-semver-release-action@v0
         with:
-          default-bump: minor
+          default-bump: patch
```

Comment:
Just for my info, does this use the same mechanism as is applied in `go-traces` -- looking for a signal in the commit message -- to override the default?

Reply:
Yes, exactly.
https://github.com/vivantehealth/go-traces/blob/main/.github/workflows/publish-tag.yml
It looks like that we need to update the publish tag action in go-traces, too.
And set the fetch-depth: 0.